### PR TITLE
Enhancing permissions table

### PR DESCRIPTION
**Pull request by matthewshaver**
## What are you changing in this pull request and why?

Enhancing the permissions tables:
Adding - to prepare for sortable tables
Adding superscript and footnotes to clarify default R permissions
Fixing spacing and reducing text

## Checklist
- [ ] I have reviewed the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) so my content adheres to these guidelines.
- [ ] The topic I'm writing about is for specific dbt version(s) and I have versioned it according to the [version a whole page](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) and/or [version a block of content](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#versioning-blocks-of-content) guidelines.
- [ ] I have added checklist item(s) to this list for anything anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."
- [ ] The content in this PR requires a dbt release note, so I added one to the [release notes page](https://docs.getdbt.com/docs/dbt-versions/dbt-cloud-release-notes).

### CODE REVIEW

File: website/snippets/_enterprise-permissions-table.md [hunk 1]
```diff
@@ -12,77 +7,93 @@ Permissions:
...
-| Metadata GraphQL API access    |       R       |               |       R         |                |   R    |
-| Permissions             |       W       |               |       W         |       W        |   R    |
-| Projects                |       W       |               |       W         |       R        |   R    |
-| Repositories            |       W       |               |       W         |                |   R    |
-| Runs                    |       W       |               |       W         |                |   R    |
-| Semantic Layer config   |       W       |               |       W         |                |   R    |
+| Environment credentials |       W       |      -        |       W         |       -        |   R    |
+| Custom env. variables   |       W       |      -        |       W         |       -        |   R    |
+| Data platform configurations|   W       |      -        |       W         |       -        |   R    |
+| Develop (IDE or dbt Cloud CLI)| W       |      -        |       W         |       -        |   -    |
```
Comment (matthewshaver) [inline website/snippets/_enterprise-permissions-table.md]:
```suggestion
| Develop (IDE or CLI)       | W       |      -        |       W         |       -        |   -    |
```

File: website/snippets/_enterprise-permissions-table.md [hunk 2]
```diff
@@ -12,77 +7,114 @@ Permissions:
...
+
+* (W)rite &mdash; Create new or modify existing. Includes `send`, `create`, `delete`, `allocate`, `modify`, and `develop`.
+* (R)ead &mdash; Can view but can not create or change any fields.
+
 #### Account permissions for account roles
+
+<SortableTable>
+
+{`
 | Account-level permission| Account Admin | Billing admin |  Manage <br></br> marketplace <br></br> apps | Project creator | Security admin | Viewer |
```
Comment (runleonarun) [inline website/snippets/_enterprise-permissions-table.md]: `<br></br>` is showing up as code in the header "Manage marketplace apps" 
```suggestion
| Account-level permission| Account Admin | Billing admin |  Manage <br></br> marketplace <br></br> apps | Project creator | Security admin | Viewer | 
```

**Review by runleonarun — COMMENTED** [review]
One small code issue but otherwise looks awesome!!

**Review by runleonarun — APPROVED** [review]
Approved for when you remove line breaks in heading!

**Pull request merged**